dotnet/package/microsoft.templates.csproj

```diff
@@ -4,6 +4,10 @@
     <PackageType>Template</PackageType>
     <Description>Templates for $(_PlatformName) platforms</Description>
     <_packagePath>$(MSBuildThisFileDirectory)..\Templates\Microsoft.$(_PlatformName).Templates\</_packagePath>
+    <LocalizeTemplates>true</LocalizeTemplates>
   </PropertyGroup>
+  <ItemGroup Condition=" '$(OS)' == 'Windows_NT' ">
+    <PackageReference Include="Microsoft.TemplateEngine.Tasks" Version="$(MicrosoftTemplateEngineTasksVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
+  </ItemGroup>
```

Comment:
I don't think this will have any effect, because we build on macOS.

Reply:
Updated, thanks!